packages/kbn-text-based-editor/src/text_based_languages_editor.test.tsx

```diff
@@ -71,22 +71,14 @@ describe('TextBasedLanguagesEditor', () => {
   });
 
   it('should  render the date info with no @timestamp found', async () => {
-    const newProps = {
-      ...props,
-      isCodeEditorExpanded: true,
```

Comment:
ℹ️ Removed the `isCodeEditorExpanded` from the tests here. It is not a valid property after the cleanup